pdf_build_src/process_markdowns.py

```diff
@@ -141,25 +142,34 @@ def modify_changelog():
         file.writelines(data)
 
 
-"""Number of chars maximal in one line approximated from a line of the PDF"""
+# Number of chars maximal in one line approximated from a line of the PDF
 NB_CHARS_LINE_PDF = 100
 
-def correct_table(table):
-    """Compute the max number of characters and dashes for each column and create the corrected table
+def correct_table(table, offset = [10, 40], debug=False):
+    """Create the corrected table.
+    It computes the number of characters maximal in each column and reformat line to make sure 
+    the first and second lines have enough dashes (in proportion) and fences anr correctly aligned 
+    for correct rendering in the generated PDF.
 
     Parameters
     ----------
     table : List of List of str
         Table content extracted from the markdown file.
 
+    offset : [x, y]
+        Offset that can be used to ajust the correction of number of dashes in the first (x) and 
+        second (y) columns by the number specified
+
+    debug : Bool
```

Comment:
```suggestion
    offset : [x, y]
        Offset that can be used to ajust the correction of number of dashes in the first (x) and 
        second (y) columns by the number specified
    debug : Bool
```

no spaces between parameter descriptions,